packages/@aws-cdk/aws-msk/lib/cluster.ts

```diff
@@ -361,6 +361,13 @@ export class ClientAuthentication {
     return new ClientAuthentication(undefined, props);
   }
 
+  /**
+   * SASL + TLS authentication
+   */
+  public static saslTls(saslProps: SaslAuthProps, tlsProps: TlsAuthProps): ClientAuthentication {
```

Comment:
Can we make this take a `SaslTlsAuthProps` object like this:

```ts
export interface SaslTlsAuthProps extends SaslAuthProps, TslAuthProps { }
```

So that we don't have a function that takes 2 object parameters?

Reply:
Addressed this in the last commit, I hope its better now